lib/defaults.js

```diff
@@ -36,6 +56,29 @@ const generateUniquePackageOrdId = (ordNamespace, name, tag, visibility) => {
     return `${ordNamespace}:package:${regexWithRemoval(name)}${tag}${visibilitySuffix}:v1`;
 };
 
+const mapEnvPackageInfo = (packageConfig) => {
+    if (!packageConfig) {
+        return { vendor: undefined };
+    }
+
+    const filteredEnv = {};
+    for (const key in packageConfig) {
+        const value = packageConfig[key];
+        if (!value) {
+            continue;
+        }
+        const checkFunction = packageTypeChecks[key];
+        if (!checkFunction) {
+            continue;
+        }
+        if (!checkFunction(value)) {
+            continue;
+        }
+        filteredEnv[key] = value;
+    }
```

Comment:
```suggestion
    for (const key in packageConfig) {
        const value = packageConfig[key];
        const checkFunction = packageTypeChecks[key];
        if (value && checkFunction && checkFunction(value)) {
            filteredEnv[key] = value;
        }
    }
```
Lets make it shorter and use positive logic as prefer